torch/_meta_registrations.py

```diff
@@ -3255,7 +3255,7 @@ def meta__int_mm(a, b):
 def meta__convert_weight_to_int4pack(w, inner_k_tiles):
     torch._check(w.dim() == 2, lambda: "w must be a 2D tensor")
     torch._check(
-        w.dtype is torch.int32,
+        w.dtype is torch.uint8,
         lambda: f"expected w to be int32, got {w.dtype}",
```

Comment:
The error message should be updated to uint8 too.

Reply:
Oh, my mistake, thanks for the notification! https://github.com/pytorch/pytorch/pull/130707